Makefile

```diff
@@ -128,16 +129,22 @@ endif
 ifeq ($(RPM_REPOSITORY),oss)
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL7}
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
+
+	mock -r epel-8-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL8}
+	mock -r epel-8-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL8}
 else
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL7}
-	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
+	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --rebuild ${RPMDIR-EL8}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
```

Comment:
Copy paste error?

Reply:
Related to your other comment. I think I actually noticed I needed to change that as well but did so on the wrong line.